lib/screens/publish_screen.dart

```diff
@@ -134,13 +136,23 @@ class _PublishScreenState extends State<PublishScreen> {
                                       fontSize: 14,
                                     ),
                           ),
-                          Text(
-                            "$kSizeText: ${provider.fileWidth} x ${provider.fileHeight}px ${provider.fileExtension.toUpperCase()}",
-                            style:
-                                Theme.of(context).textTheme.caption!.copyWith(
-                                      fontSize: 14,
-                                    ),
-                          ),
+                          if (provider.nftFormat.format != kAudioText)
```

Comment:
I am not in favor of alot of ifs. Kindly  use spread operators. This will result in all the audio related in one widget array and all the text related in another array.